picoquic/picoquic_internal.h

```diff
@@ -439,7 +439,7 @@ typedef struct st_picoquic_stream_head_t {
 #define IS_CLIENT_STREAM_ID(id) (unsigned int)(((id) & 1) == 0)
 #define IS_BIDIR_STREAM_ID(id)  (unsigned int)(((id) & 2) == 0)
 #define IS_LOCAL_STREAM_ID(id, client_mode)  (unsigned int)(((id)^(client_mode)) & 1)
-#define STREAM_ID_FROM_RANK(rank, is_server_stream, is_unidir) (((rank-1ull)<<2)|(uint64_t)((is_unidir)<<1)|(is_server_stream))
+#define STREAM_ID_FROM_RANK(rank, is_server_stream, is_unidir) ((((uint64_t)(rank)-(uint64_t)1)<<2)|(((uint64_t)is_unidir)<<1)|((uint64_t)(is_server_stream)))
```

Comment:
What is wrong with 1ull?

Reply:
I have seen systems for which "unsigned long long" is not equivalent to "uint64_t". So I would rather use the keyboard a bit more than take risks.

Reply:
You're right. I have mistaken "unsigned long long" to be equivalent to "uint64_t".